src/server/cluster/cluster_config.cc

```
@@ -195,6 +196,26 @@ optional<ClusterNodeInfo> ParseClusterNode(const JsonType& json) {
     node.port = port.value();
   }
 
+  {
+    auto health = json.at_or_null("health");
+    if (!health.is_null()) {
+      if (!health.is_string()) {
+        LOG(WARNING) << kInvalidConfigPrefix << "invalid health status for node " << json;
```

@BorysTheDev Why does this only warn, but not return an error? (same below)

From the control plane, we'd rather have an error response to the cluster config command rather than silently fail (with a warning we don't monitor)

---

We removed the most part of the errors according to your request

---

Also this code doesn't affect the stable work of Dragonfly and isn't critical

---

> We removed the most part of the errors according to your request

What request?

If the control plane ever sends invalid configuration I think it's better to just fail the request, then we'll be notified and can investigate

---

OK